src/gui/qgsquerybuilder.cpp

```diff
@@ -94,6 +96,48 @@ QgsQueryBuilder::QgsQueryBuilder( QgsVectorLayer *layer,
   layerSubsetStringChanged();
 
   lblDataUri->setText( tr( "Set provider filter on %1" ).arg( layer->name() ) );
+
+  lblProviderStorageType->setOpenExternalLinks(true);
+
+  //TODO what to put there for other providers, e.g. postgis?
+  QStringList supportedSqlDialectsText = QStringList();
+  if ( layer->dataProvider()->name() == "ogr" ) {
+
+    GDALDriverH driver = GDALGetDriverByName( layer->dataProvider()->storageType().toLocal8Bit().constData() );
```

Comment:
unless we have a strong reason to want Local8Bit, UTF-8 is generally a more robust choice (even if that context, only ASCII is really expected):
```suggestion
    GDALDriverH driver = GDALGetDriverByName( layer->dataProvider()->storageType().toUtf8().constData() );
```

Reply:
Thanks! I had simply copied that from another place in the sources without _really_ knowing what I was doing. :}